### ISSUE TITLE
Lag in character select on mister

### ISSUE THREAD

**Issue by MrsBison** [city41/kof94te#132]
While trying to select the character in the new overview character select rooster....
Joystick do not react fast. Example... click multiple times until the selection curser moves.
Also ingame plays not that smooth than on the regular 94.

I used the Mame KoF94.zip --> converted into KoF94te.neo --> Played on Mister

**Comment by city41** [comment]
Do you see white noise at the top of the screen in character select?

![image](https://github.com/city41/kof94te/assets/141159/5818aa1b-1277-411d-8e95-fb5e0efa10d2)


I have ran the hack on the mister very extensively, it's the main way I play neo these days. The hack works perfectly on my mister. But I haven't updated my neo geo core in a long time.

I suspect a newer version of the core introduced the lag. I won't be able to confirm until Wednesday or so.

**Comment by riggles1** [comment]
If you're playing on a CRT you can easily check lag by using ultra slowmotion capture mode of the camera, my old samsung s8+ has that so I suspect most phones do at this point. Making sure that the button is also recorded as it's pressed, then counting the screen flashes. Do multiple takes and compare with the vanilla game.

Example of me capturing the zapper target detection flashes: https://imgur.com/9JxrDsz

**Comment by MrsBison** [comment]
Hey...

Yes ive also noticed these weird white graphical issue at the top of the screen.
Im playing with my Mister on a LCD and all other Neo Geo games working perfect. 
Even Art of Fighting 2 runs so good that i dont notice any difference to my AES
and AoF2 is vers timing relevant due to the special moves which are programable.

Cheers.

**Comment by city41** [comment]
I can't confirm until Wednesday but I do suspect the mister core. If anyone wants to test older versions of the core I suspect you'll find one that works.

Like I said I've ran the hack on the mister very extensively, but on an old version of the core.

And I've also ran the hack extensively on a real neo geo with no issues.

There shouldn't be a need to do any lag testing, random select almost certainly runs noticably slow.

This video is a good example of how fast random select should be https://youtu.be/PK0A1jmD7K0?si=pSWPO1M_uc19h5VA

**Comment by city41** [comment]
This is due to the MiSTer's main binaries themselves, not the NeoGeo core as I suspected.

I filed an issue at the main MiSTer repo [here](https://github.com/MiSTer-devel/Main_MiSTer/issues/904).

Here is a video showing why I came to this conclusion

https://github.com/city41/kof94te/assets/141159/f31ef760-750f-4316-a217-30d7c2da0d8d

Unfortunately I don't think there is much I can do about this. I'm going to wait to see what the MiSTer team has to say. But at least for now, they only way to run the hack on a MiSTer is to use an old version. I am using v230233 and it runs beautifully on that version.
